linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -151,7 +151,10 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
     val parentVarWithGlobals = for (parentIdent <- tree.superClass) yield {
       implicit val pos = parentIdent.pos
       if (!tree.kind.isJSClass) {
-        WithGlobals(encodeClassVar(parentIdent.name))
+        if (className == ThrowableClass)
+          WithGlobals(js.VarRef(js.Ident("Error")))
```

Comment:
So just to double check: The reason we do not need to track `Error` as global here is because it is not dangerous?

Reply:
Yes: it's not dangerous, and we only use outside of method bodies.